tests/acceptance/test_update_control.py

```diff
@@ -579,7 +583,13 @@ def wait_for_state(state_to_wait_for):
             log = wait_for_state("Cleanup")
             assert "ArtifactFailure" not in log
 
+        except:
+            connection.run("journalctl -u mender-client | cat")
```

Comment:
`print(connection.run(...).stdout)` ?

Reply:
Makes  sense right, @lluiscampos ? I don't know if we need `stderr` too? :thinking:

Reply:
Fabric will already print stdout/stderr to terminal when running the command, why do we want it wrapped it in `print`? If any maybe you mean to add it to `logging` so that `Pytest` appends it on failures? (which I dislike, I prefer the failure to be about the code, and then browse up and see the test stdout)

Reply:
I tried it as a fix for the "would have blocked on write" error which happens for me when the output is very long and gets truncated. But the suggested fix didn't make any difference, I suspect it has wrapped stdout anyway, so it doesn't matter if you `print` it.

Reply:
Okey, thanks for the background. I am also having often that error and I suspected is related to Fabric too.

Anyway, if you say is not really helping, I leave it as is until we find a proper solution.